source/MissionAction.cpp

```diff
@@ -445,14 +465,18 @@ void MissionAction::Do(PlayerInfo &player, UI *ui, const System *destination, co
 		for(const auto &eit : it.second)
 			player.AddSpecialLog(it.first, eit.first, eit.second);
 	
+	for(const auto &it : giftShips)
+	{
+		DoGiftShip(player, it.first, it.second, ui);
+	}
```

Comment:
Style: unnecessary braces.
```suggestion
		DoGiftShip(player, it.first, it.second, ui);
```